package.json

```diff
@@ -14,6 +14,7 @@
         "version:release": "changelogithub",
         "format": "prettier -c .",
         "format:write": "prettier -w .",
+        "lint": "circomspect packages/*/src -L ./node_modules/circomlib/circuits",
```

Comment:
`circomspect` seems to return a non zero code even if only outputs of `WARNING` level are detected.
This is what the ci check fails.

So we definitely need to address the warning or allow them with eg:

```commandline
circomspect -a CS0006 -a CS007 -a CS0010 -a CS0014 -a CS0017 -a CS0018 packages/*/src -L ./node_modules/circomlib/circuits
```

use `circomspect -v` to see rule IDS.

Reply:
see #8 for a fix

Reply:
Thanks!